build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -0,0 +1,168 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: packetcaptures.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - jsonPath: .status.phase
+          description: The phase of the PacketCapture.
+          name: Phase
+          type: string
+        - jsonPath: .spec.source.pod
+          description: The name of the source Pod.
+          name: Source-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.pod
+          description: The name of the destination Pod.
+          name: Destination-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.ip
+          description: The IP address of the destination.
+          name: Destination-IP
+          type: string
+          priority: 10
+        - jsonPath: .spec.timeout
+          description: Timeout in seconds.
+          name: Timeout
+          type: integer
+          priority: 10
+        - jsonPath: .metadata.creationTimestamp
+          name: Age
+          type: date
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
```

Comment:
updated to use crd schema instead of webhook to do the spec validation. For now there is only one case ( reject source.pod == "" && destnation.pod == "") i couldn't find support from openAPIV3Schema so i put it back in the agent controller.  

cc @tnqn @luolanzone

Reply:
I think you can use the following:
```
openAPIV3Schema:
  type: object
  required:
    - spec
  properties:
    spec:
      type: object
      required:
        - fileServer
        - firstNCaptureConfig
        - source
        - destination
      anyOf:
        - properties:
            source:
              required: [ pod ]
        - properties:
            destination:
              required: [ pod ]
```

Reply:
Thanks. It works. code updated.